### PR TITLE
Adjust tests workflow for fork PRs

### PR DESCRIPTION
**Pull request by max-ostapenko**
- [x] changed tests workflow to `pull_request_target` trigger

Potential vulnerabilities:
- stolen secrets.

### CODE REVIEW

**Comment by max-ostapenko** [comment]
@tunetheweb, I did minimal changes (hope I understood you correct)
And existing PRs need to rebase in order to use a new workflow.

**Comment by tunetheweb** [comment]
Any ideas why it didn’t kick off a run for this PR?

File: .github/workflows/wpt-test.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Tests
 
 on:
-  pull_request:
+  pull_request_target:
     branches:
```
Comment (tunetheweb) [inline .github/workflows/wpt-test.yml:5]: I don’t think you can use this as think `pull_request_target` is against the PR branch not against `main`. Lets delete these two lines and run it for all PR branches.

Comment (tunetheweb) [reply .github/workflows/wpt-test.yml:5]: Then I’m at a loss as to why this PR hasn’t triggered this test.

Comment (tunetheweb) [reply .github/workflows/wpt-test.yml:5]: I wonder if it has to be merged to take effect?

**Comment by max-ostapenko** [comment]
I think it's because it will run once merged ([source](https://stackoverflow.com/a/77069245))
Could replicate it in [a test repo](https://github.com/max-ostapenko/test/pull/4).

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**